stl/inc/regex

```diff
@@ -739,6 +765,15 @@ _NODISCARD bool operator==(const _Iter_value_t<_BidIt>* _Left, const sub_match<_
     return _Right._Match_equal(_Left);
 }
 
+#if _HAS_CXX20
+template <class _BidIt>
+_NODISCARD auto operator<=>(const sub_match<_BidIt>& _Left, const _Iter_value_t<_BidIt>* _Right) {

```

Comment:
We have been preferring the pretty `iter_value_t` to the ugly `_Iter_value_t` in new C++20 code. Should we do so here as well, or use `_Iter_value_t` for consistency across language modes?

Reply:
Didn't know about that, sounds good.

Reply:
`iter_value_t` isn't lit up for EDG yet.